contrib/devtools/copyright_header.py

```diff
@@ -41,8 +41,8 @@ def applies_to_file(filename):
     for excluded_dir in EXCLUDE_DIRS:
         if filename.startswith(excluded_dir):
             return False
-    return ((EXCLUDE_COMPILED.match(filename) is None) and
-            (INCLUDE_COMPILED.match(filename) is not None))
+    return ((EXCLUDE_COMPILED.match(filename) is None)
+            and (INCLUDE_COMPILED.match(filename) is not None))
```

Comment:
`W504` should be disabled. This is not a critical error that should be blocking a merge